package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "diameter-avp-object",
-  "version": "1.0.6",
```

Comment:
Could you please revert this line? We bump up versions using the `npm version` command after changes are merged in. Besides, we may want to bump up the minor version for this change.

Reply:
Done, sorry, my bad.